src/component/Input/input.css

```diff
@@ -0,0 +1,34 @@
+input{
```

Comment:
Используй класс вместо тега для стилизации. И добавляй этот класс на соответсвующий компонент
```suggestion
.input{
```